hdfs3/core.py

```diff
@@ -24,7 +24,7 @@ def hdfs_conf(conf_dir=None):
     """ Load HDFS config from default locations. """
     if conf_dir == None:
         confd = os.environ.get('HADOOP_CONF_DIR', os.environ.get('HADOOP_INSTALL',
-                           '') + '/hadoop/conf')
+                           '/etc') + '/hadoop/conf')
```

Comment:
Also, if you think about it, with this PR that scenario would only trigger an exception when there is nothing running on localhost **and** you don't have anything on `/etc/hadoop` (which is the default folder in EMR, Azure, CDH and HDP on premise, and probably more), and HADOOP_CONF_DIR/HADOOP_INSTALL are not set.